axelor-talent/src/main/resources/views/JobApplication.xml

```diff
@@ -20,36 +20,65 @@
 	            <item title="Schedule Event" action="save,action-job-application-schedule-event" readonlyIf="id == null"/>
 	        </menu>
 	    </menubar>
-		<panel name="mainPanel">
-			<field name="statusSelect" colSpan="4"/>
-			<button name="hireBtn" title="Hire" onClick="save,action-job-application-confirmation,action-job-application-method-hire" colSpan="2" showIf="statusSelect == 0" />
-			<field name="employee" showIf="statusSelect == 1" />
-			<field name="hiringStage" x-order="sequence"/>
-			<panel name="jobInfoPanel" title="Job info" colSpan="12">
-				<field name="jobPosition" />
-				<field name="jobPosition.companyDepartment" />
-				<field name="responsible" />
+		<panel name="mainPanel" readonlyIf="statusSelect == 3">
+			<field name="statusSelect" widget="NavSelect" showTitle="false" colSpan="10" readonly="true"/>
+			<field name="hiringStage" colSpan="2" widget="SuggestBox" canEdit="false"/>
```

Comment:
Put colspan 8 and 4 for status and hiringStage, as it not visible in small screen